test/codeu/chat/util/MorseConverterTest.java

```diff
@@ -0,0 +1,64 @@
+package codeu.chat.util;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+/**
+ * Tests the morse converter.
+ *
+ * @author Lauren
+ *
+ */
+public class MorseConverterTest {
+
+  /**
+   * Tests a word converted to morse.
+   */
+  @Test
+  public void testWordConversion() {
+    String morse = MorseConverter.wordToMorse("hi");
+    assertTrue(morse.equals(".... .. /"));
+  }
+
+  /**
+   * Tests that more than one sentence can be translated.
+   */
+  @Test
+  public void testSentenceConversion() {
+    String paragraph = "Hi there. This is morse code. But is it? Let's check!";
+    String morse = MorseConverter.paragraphToMorse(paragraph);
+    assertTrue(morse.equals(".... .. /– .... . .–. . /– .... .. ... /"
+        + ".. ... /–– ––– .–. ... . /–.–. ––– –.. . /"
+        + "–... ..– – /.. ... /.. – /.–.. . – /... /"
+        + "–.–. .... . –.–. –.– /"));
+
+  }
+
+  /**
+   * Tests that a morse word is translated to the correct integer.
+   */
+  @Test
+  public void testSoundTranslate() {
+    String morse = ".–.–– / –//";
+    List<Integer> translate = MorseConverter.morseToSoundTranslate(morse);
+    List<Integer> expected = new ArrayList<>();
```

Comment:
I think you could write this in one line as:
`List<Integer> expected = Arrays.asList([0, 1, 0, ....])`